msgcntr/messageforums-api/src/java/org/sakaiproject/api/app/messageforums/DraftRecipient.java

```diff
@@ -0,0 +1,32 @@
+package org.sakaiproject.api.app.messageforums;
+
+/**
+ * This class represents a saved recipient for a draft message. It can represent a user, a group, a site role, a combination of those, or all participants.
+ * It can be thought of as a more compact version of MembershipItem.
+ *
+ * @author plukasew
+ */
+public interface DraftRecipient
+{
+	public static final String ALL_PARTICIPANTS_ID = "all_participants";
```

Comment:
minor comment public is redundant on an interface

Reply:
Personally, I prefer to see it stated explicitly, but this is opinionated.

Reply:
It's redundant

Reply:
It's actually not redundant. From the official Java tutorial: "The public access specifier indicates that the interface can be used by any class in any package. If you do not specify that the interface is public, then your interface is accessible only to classes defined in the same package as the interface."

Reply:
From the JLS:
https://docs.oracle.com/javase/specs/jls/se8/html/jls-9.html#jls-9.4
Every method declaration in the body of an interface is implicitly public (§6.6). It is permitted, but discouraged as a matter of style, to redundantly specify the public modifier for a method declaration in an interface.

Reply:
I see, you're talking about methods and I was talking about the interface itself. I'll keep that in mind. Originally this file was a class, and will become a class again after I revise this PR.

Reply:
👍  Thanks Paul